content/use/networking.md

```diff
@@ -55,3 +55,35 @@ More details about the use of [docker-compose](https://docs.docker.com/compose/)
 
 
 ### <i class="fa fa-database"></i> Linking database containers
+
+Here is an example of a compose file that configures a Shiny Server that can connect to a database (PostgreSQL).
+
+```yml
+services:
+  db:
+    image: postgres:13
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    expose:
+      - 5432
```

Comment:
I believe it's not necessary to expose a port(?) I believe the containers in the same compose file are assigned to the same network group and can access over the internal network.

Reply:
You are right. As far as I know, docker compose works no differently whether `expose` is set or not (at least in docker compose V1).
In other words, it is just an annotation.
However, it is not clear from the compose file which port the DB container exposes, so I think it is a good idea to specify it as an `expose` to increase readability.

Note that the following the "compose spec" does not state that `expose` are annotations...
https://github.com/compose-spec/compose-spec/blob/5260c1bc8ee5e93ce88412bb460924e9535d9080/spec.md?plain=1#L812-L822

Reply:
:+1: ok sounds good, merge when ready!